### PR TITLE
fix: services management table layout

### PR DESCRIPTION
**Pull request by frosso**
## Description
Changing the table layout on the **Settings > WooCommerce Services** page

## Issue

Fixes https://github.com/Automattic/woocommerce-services/issues/2112 (ticket also references modal layout - done separately in https://github.com/Automattic/woocommerce-services/issues/2121)

## Steps to reproduce
- Ensure you already connected "UPS"
- Go to **Settings > WooCommerce Services**
- Go to mobile viewport
- Layout behavior is fixed

Screen Shot 2020-08-06 at 3 26 52 PM
Screen Shot 2020-08-06 at 3 27 05 PM
Screen Shot 2020-08-06 at 3 27 51 PM

### CODE REVIEW

File: client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/style.scss [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 		.carrier-accounts__list-item-carrier-icon {
 			width: 48px;
-			padding-left: 24px;
+			padding-left: 16px;
```
Comment (frosso) [inline client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/style.scss:11]: Similar to https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/woocommerce-services/views/packages/style.scss#L187-L215

File: changelog.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@
 * Fix   - Fix localization issues
 * Tweak - Improve service listing readability
 * Add   - Support UPS as a carrier (beta)
+* Fix   - Services management CSS table layout
```
Comment (c-shultz) [inline changelog.txt:17]: Same comment as: https://github.com/Automattic/woocommerce-services/pull/2121#pullrequestreview-462861940

Comment (frosso) [reply changelog.txt]: Thanks, done in c2c5153950cae9c20ff959f6e2cde3988eb93afe

**Comment by harriswong** [comment]
👀

**Pull request merged**